pyftpdlib/handlers.py

```diff
@@ -1028,11 +1028,32 @@ def __init__(self, file, type):
         """
         self.file = file
         self.type = type
+        self._prev_chunk_endswith_cr = False
         if type == 'a' and os.linesep != '\r\n':
-            self._data_wrapper = lambda x: x.replace(b(os.linesep), b'\r\n')
+            self._data_wrapper = self._posix_ascii_data_wrapper
         else:
             self._data_wrapper = None
 
+    def _posix_ascii_data_wrapper(self, chunk):
+        """The data wrapper used for sending data in ASCII mode on
+        systems using a single line terminator, handling those cases
+        where CRLF ('\r\n') gets delivered in two chunks.
+        """
+        chunk = bytearray(chunk)
+        pos = 0
+        if self._prev_chunk_endswith_cr and chunk.startswith(b'\n'):
+            pos += 1
+        while True:
+            pos = chunk.find(b'\n', pos)
+            if pos == -1:
+                break
+            if chunk[pos - 1] != 13:
```

Comment:
what's 13?

Reply:
It is `ord('\r')`. I guess you'll suggest to create a constant like `CR = ord('\r')`. Am I correct?

Reply:
Yep. I would define it at the top of this method.

Reply:
This way, `ord('\r')` will be executed each time the method is invoked.

I defined CR_BYTE at the top of `handlers.py`, which I believe is better. Is this ok?

Reply:
You're right. Yeah leave it there. =)